packages/engine/src/args.rs

```diff
@@ -9,9 +9,9 @@ use crate::utils::OutputFormat;
 #[clap(setting(AppSettings::UseLongFormatForHelpSubcommand))]
 /// Run the engine.
 pub struct Args {
-    /// experiment ID
+    /// The unique identifier of the experiment
```

Comment:
Maybe we should add a hint, that this must be a valid UUIDv4?

Reply:
The type should imply it, but I appreciate that if someone really tried to run the engine binary without the CLI binary then it would be nice for the command line help to say.